packages/clay-empty-state/src/index.tsx

```diff
@@ -59,6 +59,27 @@ const ClayEmptyState = ({
 }: IProps) => {
 	const hasImg = imgSrc || imgProps;
 
+	const [error, setError] = useState(false);
+
+	const reducedMotionImage = useMemo(() => {
+		if (error) {
+			console.warn(
+				'The image url defined in `imgSrcReducedMotion` does not exist on the server. You can provide an updated url through the attribute `imgSrcReducedMotion` or set it to `{null}`.'
+			);
+
+			return null;
+		}
+
+		const dotPosition = imgSrc?.lastIndexOf('.');
+
+		if (dotPosition !== -1 && imgSrc && imgSrcReducedMotion !== null) {
+			return `${imgSrc.substr(
+				0,
+				dotPosition
+			)}_reduced_motion${imgSrc.substr(dotPosition!)}`;
```

Comment:
Maybe we can change a little bit here to avoid breaking the URL, for example when we pass a full uri we can change the host value instead of the pathanme, what we can do is try to make sure that we are changing the path of the image. We have some solutions for that here, for example:

- Convert the `imgSrc` to a `let url = new URL(imgSrc)` and we can get just the `url.pathname`
As we are doing an inference just getting the position of the dot can be unsafe, so we can do a regex after doing the pathname to get the position of the extname of the image, for example `url.pathname.match(/.(gif|png| jpeg|jpg)/)` and then we can get the index.

As it can happen that the developer only defines the pathname of the image when we use the `new URL` we have to ensure that the host exists in the string otherwise it will throw an invalid URL error, so we can check this by defining the base URL if it does not exist with `location.host`, for example:

```js
new URL(imgSrc, imgSrc.match(/http::\/\/|https::\/\//) ? undefined : `https://${location.host}`)
```